### PR TITLE
CIP-0074? | Set minPoolCost to 0

### PR DESCRIPTION
**Pull request by ADARobinHood**
([proposal latest draft](https://github.com/ADARobinHood/CIPs/tree/patch-1/CIP%20minPoolCost%200))

### CODE REVIEW

**Comment by kieransimkin** [comment]
👍👍🏻👍🏼👍🏽👍🏾👍🏿

**Comment by rphair** [comment]
@ADARobinHood as I just wrote in https://github.com/cardano-foundation/CIPs/pull/360#issuecomment-1289454182 (so I'm copying parts of the text) there's a lot of common ground between these 2 proposals as you've observed from the Cardano Forum thread where @TobiasFancee jumped in: https://forum.cardano.org/t/cip-69-set-minpoolcost-to-0/109309/37

So since the Discord has been used to the common ground between several PoS proposals including dynamically suggested variations (there were 3 or 4 already which had parameter variations like this, some even exactly the same) I would like to invite you to all the same places.  My apologies if you've already been there but I can't line up your name with any of the participants:

---

... nice to see this begin on the Cardano Forum with such enthusiastic discussion. I've recommended an upcoming Biweekly CIP Editors' meeting to have a scope of all CIPs with this theme (revisions to PoS rewards & incentives), since a few more of them including yours have come in recently. If such an agenda is confirmed I'll post it here.

The Discord where we have these meetings online ([invite](https://discord.gg/kyaTyzkBqd)) also has a `#...cip-0050...` channel which now includes multiple CIPs for PoS revisions (I've asked the admin to rename it more generally; hopefully will happen soon). I didn't see you there yet & it would be great to have you included in that discussion. It tends to have more detail & "brainstorming" than we ever see on the GitHub thread, and also more focused peer review than on the Cardano Forum. 😎

**Review by rphair — COMMENTED** [review]
BTW so we can review this properly, please rename the document currently at this pathname: `CIP 69? - Set minPoolCost to 0`
to: `README.md`
and move the file into a descriptively named subdirectory: preferably something short beginning with `CIP` but not using a number.

File: CIP 69? - Set minPoolCost to 0 [hunk 1]
```diff
@@ -0,0 +1,73 @@
+---
+CIP 69?
```
[the remark this answers is not in the document]
Comment (rphair) [reply CIP 69? - Set minPoolCost to 0:2]: In fact @ADARobinHood there is a direct naming collision with a previously written candidate https://github.com/cardano-foundation/CIPs/pull/321, so this number should be removed ASAP from all further communications about your own proposal 🤓 (cc @KtorZ I'll go to the Cardano Forum now & notify the participants there)

[the remark this answers is not in the document]
Comment (rphair) [reply CIP 69? - Set minPoolCost to 0:2]: https://forum.cardano.org/t/cip-69-set-minpoolcost-to-0/109309/42

**Review by ADARobinHood — COMMENTED** [review]
Formatting corrections

**Review by ADARobinHood — COMMENTED** [review]
Typo fix

File: CIP 69? - Set minPoolCost to 0 (hunk 1 above)
[the remark this answers is not in the document]
Comment (rphair) [reply CIP 69? - Set minPoolCost to 0]: thanks for updates; resolving this conversation...

File: CIP 69? - Set minPoolCost to 0 [hunk 2]
```diff
@@ -0,0 +1,73 @@
+---
```
[the remark this answers is not in the document]
Comment (rphair) [reply CIP 69? - Set minPoolCost to 0]: thanks for update; resolving this conversation

File: CIP minPoolCost 0/README.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+## Rationale:
+
+98% of all pools have their cost at 340 ADA/epoch or within + 10. As the price of ADA went from $0.3 to $3, almost no operators modified their cost. As the price of ADA dropped from $3 to $0.3, almost no operators modified their cost. This shows that the minPoolCost is not related to the real cost of operating a pool and the cost of a pool is no longer related to its utility.
+
+We have a large body of research accompanied by simulations showing that removing minPoolCost will increase decentralization of the network and begin incentivizing the right behavior from SPOs and the delegating community.
+
+A minPoolCost is not in Cardano's design specification. ALL published research is in favor of setting minPoolCost to 0.
+
+
+## Effects:
```
Comment (KtorZ) [inline CIP minPoolCost 0/README.md:44]:
```suggestion
### Effects:
```

File: CIP minPoolCost 0/README.md [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+
+Stake pool desirability will become a function of pledge, performance, cost, and margin instead of only popularity. Operators will need to consolidate and grow pledge while moderating fees to remain competitive with other pools on the market.
+
+SPOs define their cost as an absolute value when submitting their registration.cert. They do not reference minPoolCost. Changing minPoolCost will not change any predefined costs. Pools that wish to leave their cost as-is can do so without any input. Pools that wish to lower their cost below 340 ADA/epoch will have to submit an updated registration.cert.
+
+
+## Backwards Compatibility:
```
Comment (KtorZ) [inline CIP minPoolCost 0/README.md:51]:
```suggestion
### Backwards Compatibility:
```

File: CIP minPoolCost 0/README.md [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+
+This proposed change is fully backwards compatible and will not require a hard-fork. Adjusting parameters is 'easy' and will require no code rewrites. This change can be implemented immediately.
+
+
+## Research and References
```
Comment (KtorZ) [inline CIP minPoolCost 0/README.md:56]:
```suggestion
### Research and References
```

File: CIP minPoolCost 0/README.md [hunk 6]
```diff
@@ -0,0 +1,74 @@
...
+https://iohk.io/en/blog/posts/2018/10/29/preventing-sybil-attacks/
+
+Stake Pools in Cardano
+https://iohk.io/en/blog/posts/2018/10/23/stake-pools-in-cardano/
+
+Incentive Paper Lecture Series (Parts 1-7)
+https://www.youtube.com/playlist?list=PLFLTrdAG7xRbAqhF3Tg8BeAea7Ard-ttn
+
+The general perspective on staking in Cardano
+https://iohk.io/en/blog/posts/2020/11/13/the-general-perspective-on-staking-in-cardano/
```
Comment (KtorZ) [inline CIP minPoolCost 0/README.md:74]:
```suggestion

## Path to Active

- [ ] Reach out to IOG's core engineering and research teams for feedback regarding the feasibility and soundness of this proposal. 

- [ ] Create a protocol parameter update
    - [ ] Build the raw transaction
    - [ ] Have genesis delegates (IOG, CF & Emurgo) sign the update 
    - [ ] Submit it to the network
    - [ ] Confirm the update is eventually live and adopted by the network 
```

**Comment by KtorZ** [comment]
We discussed this proposal in today's editors meeting (number 57). As for other proposals regarding the reward sharing scheme or the ledger rule, the proposal shall be merged as `Proposed` (with the above addition) while the founding members work with the community on a framework to review and proceed with changes in those areas of the protocol (see discussion in the -- upcoming -- meeting notes)

**Review by ADARobinHood — COMMENTED** [review]
changes

File: CIP minPoolCost 0/README.md [hunk 7]
```diff
@@ -0,0 +1,74 @@
+---
+CIP: ?
+Title: Set minPoolCost to 0
+Author: Robin of Loxley <adarobinhood@tutanota.com>
+Comments-URI: https://forum.cardano.org/t/cip-69-set-minpoolcost-to-0/109309
+Status: Draft
```
Comment (KtorZ) [inline CIP minPoolCost 0/README.md:6]:
```suggestion
Status: Proposed
```

**Review by KtorZ — CHANGES_REQUESTED** [review]

**Review by ADARobinHood — COMMENTED** [review]
+

**Review by KtorZ — APPROVED** [review]

**Comment by KtorZ** [comment]
(@ADARobinHood can you also adjust the folder name to 'CIP-0073' 🙏)

**Review by rphair — APPROVED** [review]
All in order as per yesterday's CIP meeting resolution & the new Path to Active.

**Pull request merged**